examples/demo_all_cisco.pp

```diff
@@ -49,6 +49,7 @@
   include ciscopuppet::cisco::demo_stp_vlan
   include ciscopuppet::cisco::demo_tacacs_server
   include ciscopuppet::cisco::demo_tacacs_server_host
+  include ciscopuppet::cisco::demo_upgrade
```

Comment:
I would comment this out in the all file so that it does not get run by default.